### ISSUE TITLE
base names are inconsistent in armies

### ISSUE THREAD

**Issue by marcpawl** [leberechtreinhold/dba3_tts#21]
Mounted troop names are inconsistent using hyphen, underscore, and colon.  Mtd-4Bw vs 4Bw-Mtd.
Page 7, has the mounted in the front. Mtd-X.
2Cm should be LCm
2LH should be LH
3Cv should be Cv
Cp should be CP

  'Art', 'El', 
  '2LH', 'LH', 'LCm', 'Cm', '2Cm', 'SCh', 'LCh', 'HCh', '3Cv', 
  'Cv', '6Cv', '3Kn', '4Kn', '6Kn', 
  '3Ax','4Ax',
  '3Bd', '6Bd', '4Bd', 
  '3Bw','4Bw', '8Bw', 
  '3Bw_Mtd', '3Bw-Mtd', '3Mtd-Bw', '3Bw:Mtd',
  '4Bw-Mtd', 'Mtd-4Bw', '4Bw_Mtd', 
  '3Cb', '4Cb', '8Cb', '4Cb_Mtd', 
  '3Lb', '4Lb', '4Lb_Mtd', '4Lb-Mtd', '8Lb',
  '5Hd', '7Hd',
  '3Pk', '4Pk',
  'Ps',
  'Sp', '8Sp',
  '3Wb','4Wb',
  'WWg',
  'Cp', 'CP', '3CP', 'Lit', 'CWg',
  'Camp'

**Comment by marcpawl** [comment]
basement:data marcpawl$ grep  -e 2Cm -e Mtd *arm* | grep name | sort -u
data_armies_book_I.ttslua:        name = '3Bw_Mtd',
data_armies_book_I.ttslua:        name = '3Bw_Mtd_Gen',
data_armies_book_II.ttslua:        name = '2Cm_Gen',
data_armies_book_III.ttslua:        name = '3Bw-Mtd',
data_armies_book_IV.ttslua:        name = '3Bw:Mtd',
data_armies_book_IV.ttslua:        name = '3Bw_Mtd',
data_armies_book_IV.ttslua:        name = '3Mtd-Bw',
data_armies_book_IV.ttslua:        name = '4Bw_Mtd_Gen',
data_armies_book_IV.ttslua:        name = '4Cb_Mtd',
data_armies_book_IV.ttslua:        name = '4Lb-Mtd',
data_armies_book_IV.ttslua:        name = '4Lb_Mtd',
data_armies_book_IV.ttslua:        name = '4Lb_Mtd_Gen',
data_armies_book_IV.ttslua:        name = 'Mtd-4Bw',

**Comment by leberechtreinhold** [comment]
Yup, you are completely right, what a mess particularly the Mtd. The 2Cm is caused because that's the name they had in DBA2.2 and probably carried over, but it should be LCm. I don't even know what happened with the Mtd but it's obviously very wrong, it shoudl always be Mtd-TroopType, like Mtd-4Bw.

I will fix them tomorrow, thanks!
